src/Invoker.php

```diff
@@ -79,4 +94,15 @@ public function handle(
             ]);
         }
     }
+
+    /**
+     * Static registry for declaring functions
```

Comment:
nit: let's clarify that this is internal

```suggestion
     * Static registry for declaring functions. Internal only.
     *
     * @internal
     *
```
```